.gitmodules

```diff
@@ -7,3 +7,9 @@
 [submodule "lib/i18next"]
 	path = lib/i18next
 	url = https://github.com/i18next/i18next.git
+[submodule "lib/bootstrap"]
+	path = lib/bootstrap
+	url = https://github.com/twbs/bootstrap-sass/
+[submodule "lib/magnific-popup"]
+	path = lib/magnific-popup
+	url = https://github.com/dimsemenov/Magnific-Popup
```

Comment:
Starting with 7c3e4119ddeb89b822f6c73d2c9fd38281ac32ac I decided to follow your proposal , because I saw how many libs are laying around in the lib folder. So new libs will now be added through bower and old libs will be converted to bower after 2.1 was released. But this will not cause that we always use the original project, because not every project accepts pull requests (or  accepts pull requests in a reasonable time).